src/Tools/Md2Html/Md2Html.cs

```diff
@@ -97,8 +97,16 @@ private static void ConvertRelativeLocalImagePathsToAbsolute(string mdFilePath,
         /// <returns>return sanitized content string or an empty string if no sanitizing happened</returns>
         internal static string Sanitize(string content)
         {
+            string output;
             HtmlSanitizer.ContentWasUpdated = false;
-            var output = HtmlSanitizer.Sanitize(content);
+            if (content.Contains(@"<body "))
+            {
+                output = HtmlSanitizer.SanitizeDocument(content);
```

Comment:
whats the difference between these two?

Reply:
This one sanitize a document, e.g. you have a body with an optional head. The other one is for sanitizing fragments. head is ignored for fragments and that was the problem here.